product-service/getProductsList.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+import products from './products.json';
+import { headers } from './constants';
+
+export const get = async event => {
+  return {
+    headers,
+    statusCode: 200,
+    body: JSON.stringify(products),
```

Comment:
Potentially JSON.stringify may throw an error, so I'd recommend to wrap it in try-catch too.

Reply:
Added